fuse/baton_rouge_da.py

```diff
@@ -0,0 +1,44 @@
+import sys
+
+sys.path.append("../")
+from lib.path import data_file_path
+from lib.columns import rearrange_allegation_columns
+from lib.personnel import fuse_personnel
+from lib.post import load_for_agency
+import pandas as pd
+from lib import events
+
+
+def fuse_events(post):
+    builder = events.Builder()
+    builder.extract_events(
+        post,
+        {
+            events.OFFICER_LEVEL_1_CERT: {
+                "prefix": "level_1_cert",
+                "parse_date": "%Y-%m-%d",
+                "keep": ["uid", "agency"],
+            },
+            events.OFFICER_PC_12_QUALIFICATION: {
+                "prefix": "last_pc_12_qualification",
+                "parse_date": "%Y-%m-%d",
+                "keep": ["uid", "agency"],
+            },
+            events.OFFICER_HIRE: {"prefix": "hire", "keep": ["uid", "agency"]},
+        },
+        ["uid"],
+    )
+    return builder.to_frame()
+
+
+if __name__ == "__main__":
+    brady = pd.read_csv(data_file_path("clean/cprr_baton_rouge_da_2021.csv"))
+    agency = brady.agency[0]
+    post = load_for_agency("clean/pprr_post_2020_11_06.csv", agency)
+    per = fuse_personnel(brady, post)
```

Comment:
I saw 3 personnel datasets for Baton Rouge DA. Why do you take roster data from Brady and POST instead?

Reply:
The officers listed on a brady list aren't employed by the DA so they wont' show up on the personnel dataset. The brady list is a list created by the DA office of officers, who are employed either by a PD or SO.

Reply:
Aren't we supposed to save officers who are employed by the DA in the file `fuse/per_baton_rouge_da.csv`?